sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -161,6 +167,26 @@ export class JsonRpcProvider {
 		return requestSuiFromFaucet(this.connection.faucet, recipient, httpHeaders);
 	}
 
+	async requestSuiFromFaucetV1(
+		recipient: SuiAddress,
+		httpHeaders?: HttpHeaders,
+	): Promise<BatchFaucetResponse> {
+		if (!this.connection.faucetV1) {
+			throw new Error('Faucet URL is not specified');
+		}
+		return requestSuiFromFaucetV1(this.connection.faucetV1 + 'gas', recipient, httpHeaders);
+	}
+
+	async getRequestStatus(
```

Comment:
I think this name is way too generic, I'd rename it so that it's obvious this is related to faucet requests.